### PR TITLE
Add documentation for VeSync humidifiers

### PR DESCRIPTION
**Pull request by jspayd**
## Proposed change

Adds documentation for VeSync humidifiers

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/62907
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @jspayd, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#20901).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/vesync.markdown [hunk 1]
```diff
@@ -99,6 +109,33 @@ VeSync air purifiers will expose the following details depending on the features
 | `night_light`           | The current status of the night light (Core200S)                                  | off             |
 | `child_lock`            | The current status of the child lock (Core200S)                                   | off             |
 
+## Humidifier Exposed Attributes
+
+VeSync air purifiers will expose the following details depending on the features supported by the model:
+
+| Attribute                     | Description                                                                       | Example             |
+| ----------------------------- | --------------------------------------------------------------------------------- | ------------------- |
+| `humidity`                    | The target humidity of the humidifier.                                            | 40                  |
```
Comment (frenck) [inline source/_integrations/vesync.markdown]: I think we should avoid using tables, as they are hard to read on mobile devices.

Comment (jspayd) [reply source/_integrations/vesync.markdown]: Is avoiding tables more important than consistency on the same page? The docs for all other devices on this page use a table of this form for exposed attributes.

**Comment by jspayd** [comment]
Rebased and updated for changes I made to the integration

File: source/_integrations/vesync.markdown [hunk 2]
```diff
@@ -111,6 +121,44 @@ VeSync air purifiers will expose the following details depending on the features
...
+
+VeSync humidifiers have settings other than just a target humidity, like a manual mist level and an on/off display toggle.
+These are exposed as configuration entities alongside the humidifier itself.
+
+| Entity ID                                 | Description                                    |
+| ----------------------------------------- | ---------------------------------------------- |
+| `number.<humidifier name>_mist_level`     | The target mist level of the humidifier        |
+| `light.<humidifier name>_night_light`     | The brightness of the humidifier's night light |
+| `switch.<humidifier name>_display`        | The humidifier's display                       |
+| `switch.<humidifier name>_automatic_stop` | The humidifier's automatic stop feature        |
```
Comment (frenck) [inline source/_integrations/vesync.markdown]: This integration uses config entries, and entity IDs are not guaranteed and thus should not be mentioned like this. 

Instead, name and describe the sensor it creates.

File: source/_integrations/vesync.markdown (hunk 1 above)
Comment (frenck) [reply source/_integrations/vesync.markdown]: > Is avoiding tables more important than consistency on the same page?

Make it consistent without table than 🤷

Comment (jspayd) [reply source/_integrations/vesync.markdown]: I'm confused. Are you suggesting I convert all existing tables in this file into some other form? That's the only way I could avoid a table here *and* make it consistent with the rest of the page.

Comment (frenck) [reply source/_integrations/vesync.markdown]: >  Are you suggesting I convert all existing tables in this file into some other form?

Yes, that is what I was suggesting.

Comment (jspayd) [reply source/_integrations/vesync.markdown]: IMO that’s out of the scope of my feature addition, but I’ll convert the tables in this PR into some other form so I’m at least not creating more technical debt.

Comment (jspayd) [reply source/_integrations/vesync.markdown]: I went ahead and converted the new tables to lists, and also replaced the entity IDs with generic names. Lmk if there's anything else that needs to be changed.

**Comment by frenck** [comment]
Closing this PR as the parent PR has been closed.

**Pull request closed without merging**